### PR TITLE
REF: TransactionStatus Hooks

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6jfkcvkktnajjw84z2vzj70q4m

[download apk](http://filestorage.bluewallet.io:1488/1606707508.apk)

**Review by limpbrains — APPROVED** [review]

**Comment by ncoelho** [comment]
No bump/cancel buttons appeared on making a transaction

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/kfeqdkcpvgtv1ngk3t7x3gctmr

[download apk](http://filestorage.bluewallet.io:1488/1606756634.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/271rgvxaqr77m3cq8c2q8ck90c

[download apk](http://filestorage.bluewallet.io:1488/1606791656.apk)

**Review by Overtorment — CHANGES_REQUESTED** [review]
pls double check that RBF & CPFP work
otherwise lgtm

**Comment by ncoelho** [comment]
Screenshot 2020-12-01 at 16 38 39
Screenshot 2020-12-01 at 16 38 29

**Review by ncoelho — CHANGES_REQUESTED** [review]
Can't RBF

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6n95wxvwpbx4p3ytqyjra5mp28

[download apk](http://filestorage.bluewallet.io:1488/1606887952.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/u79369a18988mzzxjw3rxnjuyc

[download apk](http://filestorage.bluewallet.io:1488/1606888804.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/q4f0n3xkq0hnjtuc552urr5g6r

[download apk](http://filestorage.bluewallet.io:1488/1606942584.apk)

**Review by Overtorment — APPROVED** [review]
lgtm. tested RBG & CPFP on testflight build, worked as expected. @ncoelho your turn

File: screen/transactions/transactionStatus.js [hunk 1]
```diff
@@ -11,22 +11,350 @@ import {
...
+  const { setSelectedWallet, wallets, txMetadata, getTransactions } = useContext(BlueStorageContext);
+  const [isHandOffUseEnabled, setIsHandOffUseEnabled] = useState(false);
+  const { hash } = useRoute().params;
+  const { navigate, setOptions } = useNavigation();
+  const { colors } = useTheme();
+  const wallet = useRef();
+  const [isCPFPPossible, setIsCPFPPossible] = useState();
+  const [isRBFBumpFeePossible, setIsRBFBumpFeePossible] = useState();
+  const [isRBFCancelPossible, setIsRBFCancelPossible] = useState();
+  const [tx, setTX] = useState();
```
Comment (limpbrains) [inline screen/transactions/transactionStatus.js:40]: Same here, I believe `useMemo` is better that `useRef` for `wallet` and `tx`. But that's not really important

**Review by limpbrains — APPROVED** [review]

**Review by ncoelho — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**